posthog/warehouse/data_load/service.py

```diff
@@ -56,6 +56,12 @@ def get_sync_schedule(external_data_schema: "ExternalDataSchema"):
             asdict(inputs),
             id=str(external_data_schema.id),
             task_queue=str(DATA_WAREHOUSE_TASK_QUEUE),
+            retry_policy=RetryPolicy(
+                initial_interval=timedelta(seconds=10),
+                maximum_interval=timedelta(seconds=60),
+                maximum_attempts=3,
+                non_retryable_error_types=["NondeterminismError"],
```

Comment:
Going by the temporal docs, this _should_ be on the workflow decorator or on the worker, not in the retry policy. But maybe it can go here too, and this was omitted. We can try.

Reply:
If it can go in the workflow decorator, I would assume it can go here too - considering they're the same settings! But yeah, lets give it a go